### PR TITLE
Kafka + SQL storage variant re-using datamodel from Streams storage variant

### PR DESCRIPTION
**Pull request by famartinrh**
I started from the source at #1000 but re-implemented it to use the datamodel we use in the streams storage variant.

This means, now we put into kafka our datamodel, that represents the actions to be performed in to database(create artifact, update artifact,...)

At first I thought it will be necessary to have a low level API to write to the SQL storage, but I was able to do it without it, so maybe this impl is good....

### CODE REVIEW

File: storage/ksql/src/main/java/io/apicurio/registry/storage/impl/ksql/TestTypes.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apicurio.registry.storage.impl.ksql;
+
+/**
+ * @author eric.wittmann@gmail.com
+ */
+public class TestTypes {
```
Comment (alesj) [inline storage/ksql/src/main/java/io/apicurio/registry/storage/impl/ksql/TestTypes.java]: A leftover ...

**Comment by EricWittmann** [comment]
@famartinrh How about I create an upstream branch for the KSQL implementation and then you can open a PR against that instead of master?  That way it will be easier to see the differences.

**Comment by EricWittmann** [comment]
https://github.com/Apicurio/apicurio-registry/tree/learning/kafka-sql

**Comment by famartinrh** [comment]
sure, I'm going to create the new PR now

**Pull request closed without merging**